banyand/measure/merger.go

```diff
@@ -30,6 +30,23 @@ import (
 	"github.com/apache/skywalking-banyandb/pkg/watcher"
 )
 
+// mergePolicy is a global policy for all merge works.
+var mergerPolicy = NewDefaultMergePolicy()
```

Comment:
Please remove the global variable `mergerPolicy` and instead add it as a field of the `tstable`.

Reply:
I cannot understand. The policy should be a global one right? We wouldn't have different policies for different tables?